RecoTauTag/RecoTau/plugins/PATTauDiscriminationByMVAIsolationRun2.cc

```diff
@@ -0,0 +1,306 @@
+      mvaInput_[8]  = std::min((float)10., std::fabs(tau->dxy_Sig()));
+      mvaInput_[9]  = ( tau->hasSecondaryVertex() ) ? 1. : 0.;
+      mvaInput_[10] = std::sqrt(decayDistMag);
+      mvaInput_[11] = std::min((float)10., tau->flightLengthSig());
+    } else if ( mvaOpt_ == kDBoldDMwLT || mvaOpt_ == kDBnewDMwLT ) {
+      mvaInput_[0]  = std::log(std::max((float)1., (float)tau->pt()));
+      mvaInput_[1]  = std::fabs(tau->eta());
+      mvaInput_[2]  = std::log(std::max((float)1.e-2, chargedIsoPtSum));
+      mvaInput_[3]  = std::log(std::max((float)1.e-2, neutralIsoPtSum));
+      mvaInput_[4]  = std::log(std::max((float)1.e-2, puCorrPtSum));
+      mvaInput_[5]  = std::log(std::max((float)1.e-2, photonPtSumOutsideSignalCone));
+      mvaInput_[6]  = tauDecayMode;
+      mvaInput_[7]  = std::min((float)30., nPhoton);
+      mvaInput_[8]  = std::min((float)0.5, ptWeightedDetaStrip);
+      mvaInput_[9]  = std::min((float)0.5, ptWeightedDphiStrip);
+      mvaInput_[10] = std::min((float)0.5, ptWeightedDrSignal);
+      mvaInput_[11] = std::min((float)0.5, ptWeightedDrIsolation);
+      mvaInput_[12] = std::min((float)100., leadingTrackChi2);
+      mvaInput_[13] = std::min((float)1., eRatio);
+      mvaInput_[14]  = TMath::Sign((float)+1., tau->dxy());
+      mvaInput_[15]  = std::sqrt(std::min((float)1., std::fabs(tau->dxy())));
+      mvaInput_[16]  = std::min((float)10., std::fabs(tau->dxy_Sig()));
+      mvaInput_[17]  = TMath::Sign((float)+1., tau->ip3d());
+      mvaInput_[18]  = std::sqrt(std::min((float)1., std::fabs(tau->ip3d())));
+      mvaInput_[19]  = std::min((float)10., std::fabs(tau->ip3d_Sig()));
+      mvaInput_[20]  = ( tau->hasSecondaryVertex() ) ? 1. : 0.;
+      mvaInput_[21] = std::sqrt(decayDistMag);
+      mvaInput_[22] = std::min((float)10., tau->flightLengthSig());
+    } else if ( mvaOpt_ == kPWoldDMwLT || mvaOpt_ == kPWnewDMwLT ) {
+      mvaInput_[0]  = std::log(std::max((float)1., (float)tau->pt()));
+      mvaInput_[1]  = std::fabs(tau->eta());
+      mvaInput_[2]  = std::log(std::max((float)1.e-2, chargedIsoPtSum));
+      mvaInput_[3]  = std::log(std::max((float)1.e-2, neutralIsoPtSum));
+      mvaInput_[4]  = std::log(std::max((float)1.e-2, footprintCorrection));
+      mvaInput_[5]  = std::log(std::max((float)1.e-2, photonPtSumOutsideSignalCone));
+      mvaInput_[6]  = tauDecayMode;
+      mvaInput_[7]  = std::min((float)30., nPhoton);
+      mvaInput_[8]  = std::min((float)0.5, ptWeightedDetaStrip);
+      mvaInput_[9]  = std::min((float)0.5, ptWeightedDphiStrip);
+      mvaInput_[10] = std::min((float)0.5, ptWeightedDrSignal);
+      mvaInput_[11] = std::min((float)0.5, ptWeightedDrIsolation);
+      mvaInput_[12] = std::min((float)100., leadingTrackChi2);
+      mvaInput_[13] = std::min((float)1., eRatio);
+      mvaInput_[14]  = TMath::Sign((float)+1., tau->dxy());
+      mvaInput_[15]  = std::sqrt(std::min((float)1., std::fabs(tau->dxy())));
```

Comment:
here and on L276 the code is different from 90X
```  mvaInput_[15] = std::sqrt(std::fabs(std::min((float)1., std::fabs(tau->dxy()))));```
even though the fabs is redundant, the change should be made in 90X as well for consistency